jamil/helloworld/src/main/java/br/pucrs/helloworld/api/HelloWorld.java

```diff
@@ -0,0 +1,15 @@
+package br.pucrs.helloworld.api;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import reactor.core.publisher.Mono;
+
+@RestController
+public class HelloWorld{
+ 
+@GetMapping("/hello")
+ public Mono<String> helloworld(){
```

Comment:
Usar camel case para nomes de métodos.
```suggestion
 public Mono<String> helloWorld(){
```